pkg/apis/pingcap/v1alpha1/tidbcluster_component.go

```diff
@@ -118,6 +118,8 @@ func (a *componentAccessorImpl) PodManagementPolicy() apps.PodManagementPolicyTy
 	// unified podManagementPolicy check to avoid check everywhere
 	if policy == apps.OrderedReadyPodManagement {
 		return apps.OrderedReadyPodManagement
+	} else if policy == apps.ParallelPodManagement {
```

Comment:
This is not necessary, right？

Reply:
Yes, if we use `enum` then this is not necessary. I have removed it in the latest commit.